sdk/core/Azure.Core/src/Shared/LruCache.cs

```diff
@@ -1,17 +1,20 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System.Collections;
 using System.Collections.Generic;
 
-namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro
+#nullable enable
+
+namespace Azure.Core
 {
     /// <summary>
     /// A simple LRU cache implementation using a doubly linked list and dictionary.
     /// </summary>
     /// <typeparam name="TKey">The type of key</typeparam>
     /// <typeparam name="TValue">The type of value</typeparam>
     internal class LruCache<TKey, TValue> : IEnumerable<KeyValuePair<TKey, TValue>>
```

Comment:
Would it be possible to move this type into Azure.Core.Experimental with the SchemaValidator type as well?  We can make it public there for now, and consider whether to take it into internal shared source as part of moving SchemaValidator into Azure.Core.  Thanks!

Reply:
Yes! Will update